es6 & beyond/ch4.md

```diff
@@ -118,16 +118,15 @@ ajax( "http://some.url.1" )
 	}
 )
 .then( function fulfilled(contents){
-	// `contents` comes from the subsequent
-	// `ajax(..)` call, whichever it was
+	// `contents` vem de uma das chamadas subsequentes de `ajax(..)`
 } );
 ```
 
-It's important to note that an exception (or rejected promise) in the first `fulfilled(..)` will *not* result in the first `rejected(..)` being called, as that handler only responds to the resolution of the first original promise. Instead, the second promise, which the second `then(..)` is called against, receives that rejection.
+É importante notar que uma exceção (ou uma promise rejeitada) no primeiro `fulfilled(..)` *não* resultará  na primeira execução do método `rejected(..)`, já que esse manipulador responde apenas a resolução da promise original. Ao invés, a segunda promise, cujo segundo `then(..)` é chamado, recebe a rejeição.
 
-In this previous snippet, we are not listening for that rejection, which means it will be silently held onto for future observation. If you never observe it by calling a `then(..)` or `catch(..)`, then it will go unhandled. Some browser developer consoles may detect these unhandled rejections and report them, but this is not reliably guaranteed; you should always observe promise rejections.
+Nesse snippet anterior, nós não estamos ouvindo a rejeição, significando que será silenciosamente guardada para futuras observações. Se você não lidar com a rejeição utilizando `then(..)` ou `catch(..)`, então ela não será processada. Os consoles de alguns browsers podem detectar essas rejeições não processadas e reporta-las, mas isso não é garantido; you deveria sempre observar a rejeição da promise.
```

Comment:
~~Nesse snippet~~ **No trecho de código** anterior, não estamos ouvindo a rejeição, ~~significando~~ **o que significa** que **ela** será silenciosamente guardada para futuras observações. Se você não lidar com a rejeição utilizando `then(..)` ou `catch(..)`, ~~então~~ ela não será processada. Os consoles de alguns ~~browsers~~ **navegadores** podem detectar essas rejeições não processadas e reporta-las, mas isso não é garantido; ~~you~~ **você** deveria sempre observar a rejeição ~~da~~ **de uma** promise.